release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -88,6 +88,22 @@ The xref:../support/remote_health_monitoring/using-insights-to-identify-issues-w
 [id="ocp-4-7-storage"]
 === Storage
 
+[id="ocp-4-5-persistent-storage-csi-ebs"]
+==== Persistent storage using the GCP PD CSI Driver Operator (Technology Preview)
+
+You can now use the Container Storage Interface (CSI) to deploy the CSI driver you need for provisioning Google Cloud Platform (GCP) persistent disks (PD). This Operator is in Technology Preview.
+
+[id="ocp-4-7-storage-vsphere-problem-detector"]
+==== vSphere Problem Detector Operator
+
+The vSphere Problem Detector Operator periodically checks functionality of {product-title} clusters installed in a vSphere environment. The vSphere Problem Detector Operator is installed by default by the Cluster Storage Operator, allowing you to quickly identify and troubleshoot common storage issues, such as configuration and migration, on vSphere clusters.
+
+[id="ocp-4-7-storage-csi-snapshots"]
+==== Persistent storage using CSI snapshots is generally availabile
+
+You can use the Container Storage Interface (CSI) to create, restore, and delete a volume snapshot. This feature was previously introduced as a Technology Preview feature in {product-title} 4.4 and is now generally available and enabled by default in {product-title} 4.7.
```

Comment:
Should we mention that this is only for drivers that support the functionality?

Reply:
Yes, that is a good idea. 😀  I've added wording to this effect.